CovertInfiltration/Src/CovertInfiltration/Classes/X2StrategyElement_DefaultActivityChains.uc

```diff
@@ -392,9 +394,47 @@ static function X2DataTemplate CreateSupplyInterceptionTemplate()
 	
 	Template.Stages.AddItem('Activity_SupplyRescue');
 
+	Template.PostStageSetup = AttachResCon;
+
 	return Template;
 }
 
+static function AttachResCon(XComGameState NewGameState, XComGameState_Activity ActivityState)
```

Comment:
Please don't use shortened names. There is no performance penalty for long, descriptive function names, but there is a cognitive overhead when trying to understand the shortened version